best-practices/puppet_prs.md

```diff
@@ -70,6 +70,7 @@ Now you have the ability to do a test puppet run. This will run the puppet agent
 ```
 $ puppet agent --test --noop --environment=sunetidFeature
 ```
+Note: "environment" is the name of your puppet branch, and is not related to a Rails environment.
```

Comment:
Suggestion: The value of "environment" is the name ...

Reply:
updated